pkg/webservice/routes.go

```diff
@@ -98,6 +98,12 @@ var webRoutes = routes{
 		"/ws/v1/partition/:partition/queues",
 		getPartitionQueues,
 	},
+	route{
+		"Scheduler",
+		"GET",
+		"/ws/v1/partition/:partition/queue/:queue/single/:single",
```

Comment:
It seems to me the behavior of "single" is to include/exclude some fields, right? if so, could we replace the "single" by "exclude/include" query? for example, user can receive thin response by sending `/ws/v1/partition/:partition/queue/:queue?exclude=children`. Also, the "exclude/include" can be implemented to other APIs to have consistent behavior and flexible response.

Reply:
`subtree` is a clearer option than exclude or include. I cannot tell what to exclude or include. Could point to applications or maybe resources. The `subtree` refers back to the same object and we get the subtree rooted at the requested point.